start.sh

```diff
@@ -7,9 +7,9 @@ APP_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 if [[ -z "${MESOS_SLAVE_PID}" ]]; then
     source ~/.chs_env/private_env
     source ~/.chs_env/global_env
-    source ~/.chs_env/lfp-pay-api/env
+    source ~/.chs_env/penalty-payment-api/env
 
-    PORT="${LFP_PAY_API_PORT}"
+    PORT="${PPS_PAY_API_PORT}"
```

Comment:
Is this definitely correct, it's not PENALTY_PAYMENT_API_PORT?

Reply:
All env variables are renamed to PPS. Do we need to change that all to PENALTY_PAYMENT?
![Screenshot 2024-11-27 at 14 46 01](https://github.com/user-attachments/assets/5ce1e14f-77ba-4b0e-a158-7d9d286e5d04)

Reply:
It's fine to go with PPS, just wondering if where this is picked up from is set as well, did spot this on a recent search; https://github.com/companieshouse/chs-configs/blob/50e32533390834652f14373f433dd769239474e7/vagrant-chs-development/global_env#L62

Reply:
The README here currently says `PENALTY_PAYMENT_WEB_PORT`  :) https://github.com/companieshouse/penalty-payment-web/blame/master/README.md#L27

Reply:
> It's fine to go with PPS, just wondering if where this is picked up from is set as well, did spot this on a recent search; https://github.com/companieshouse/chs-configs/blob/50e32533390834652f14373f433dd769239474e7/vagrant-chs-development/global_env#L62

Regarding vagrant-chs-development, I had chat with Matt and he told that before switching to docker they were using vagrant. We don't need to make any change in vagrant-chs-development.

Reply:
> The README here currently says `PENALTY_PAYMENT_WEB_PORT` :) https://github.com/companieshouse/penalty-payment-web/blame/master/README.md#L27

I can change this to PPS as part of terraform changes. Is that fine?